news/news.md

```diff
@@ -3,6 +3,26 @@ All issues and bug reports must be opened at [GitHub](https://github.com/pymedus
 
 <br/>
 
+#### 2018-08-10 ####
+
+## Prevent errors when using Medusa with service restarters
+
+**TL;DR:**  
+Are you using a service restarter for Medusa, the Medusa Docker container, or the Windows installer?  
```

Comment:
No no no. We need to just fix the docker container. This should be a forced option.